index.html

```diff
@@ -432,7 +432,7 @@ <h3>What other people think</h3>
                                         Arthur Reeder
                                     </a>
                                 </p>
-                                <p class="user-statement">NewPipe is one of the best applications on mobile today.</p>
+                                <p class="user-statement">NewPipe is one of the best apps on mobile today.</p>
```

Comment:
~~mobile~~ Android?

Reply:
Qualifying there implies there are better apps on other OSes, and that isn't the intention.